src/win/process.c

```diff
@@ -289,6 +315,27 @@ static WCHAR* path_search_walk_ext(const WCHAR *dir,
                                  L"exe", 3,
                                  cwd, cwd_len);
   if (result != NULL) {
+    *isScript = FALSE;
+    return result;
+  }
+
+  /* Try .cmd extension */
```

Comment:
This is fine for now but eventually it should honor the `PATHEXT` environment variable.